addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -289,7 +289,7 @@ public AutomationPlan loadPlan(File f) throws IOException {
         return new AutomationPlan(this, f);
     }
 
-    public AutomationPlan loadPlan(InputStream in) throws AutomationJobException {
+    public AutomationPlan loadPlan_IsUnused(InputStream in) throws AutomationJobException {
```

Comment:
this method seems unused and is also largely identical with the constructor of `AutomationPlan`. 

might this be used somewhere from the outside? If yes, I'd have to replace the `jobData` in here as well, but if not, could we delete it?